File: Example/java/CmdHello.java
/**
 * @author Oribuin
 */
public class CmdHello extends ArsenticCommand {
    public CmdHello(Main plugin) {
        super(plugin, "hello");
    }

    @Override
    public void executeCommand(@NotNull CommandSender sender, @NotNull String[] args) {
        sender.sendMessage(HexUtils.colorify("Hello " + sender.getName() + "!"));
    }

    @Override
    public List<String> tabComplete(@NotNull CommandSender sender, @NotNull String[] args) {
        return null;
    }
}

File: Example/kotlin/CmdHello.kt
/**
 * @author Oribuin
 */
class CmdHello(plugin: Main) : ArsenticCommand(plugin, "hello") {
    override fun executeCommand(sender: CommandSender, args: Array<String>) {
        sender.sendMessage(colorify("Hello " + sender.name + "!"))
    }

    override fun tabComplete(sender: CommandSender, args: Array<String>): List<String>? {
        return null
    }
}